src/RimDev.Stuntman.AspNetCore/common/StuntmanOptions.cs

```diff
@@ -25,6 +25,19 @@ public partial class StuntmanOptions
         }
 
         /// <summary>
+        /// Determines whether bearer token authentication is enabled
+        /// when calling `UseStuntman`.
+        /// </summary>
+        public bool AllowBearerTokenAuthentication { get; set; } = true;
+
+        /// <summary>
+        /// Determines whether the UI-driven authentication is enabled
+        /// when calling `UseStuntman`. This also determines whether cookie authentication is enabled
+        /// since the two are related.
+        /// </summary>
+        public bool AllowUiAuthentication { get; set; } = true;
```

Comment:
Thinking about the naming. Don't hate it, don't love it.

`AllowCookieAuthentication` may be clearer.